frontend/src/utils/cleanProps.js

```diff
@@ -0,0 +1,100 @@
+export const cleanProps = prop => {
+  if (!styledSystemProps.includes(prop) && prop !== 'as') return true
+  return false
+}
+
+export const styledSystemProps = [
```

Comment:
We'll have to remember to update this when `styled-system` changes.

Reply:
the good news is the actual list of props is very unlikely to change